### PR TITLE
chore: clone last 10 commits

### PR DESCRIPTION
**Pull request by DanielRuf**
This PR increases the clone depth to 10 like advised in #3301.

### CODE REVIEW

**Comment by DanielRuf** [comment]
> Please check your email @DanielRuf ;)

I'm currently at work so I guess you sent it to my private email address.
Currently the time here is 12:32 pm (midday), in about 4 hours I will leave work and be at home at around 5-7 pm, so I can check my email then at home.

**Comment by ntwb** [comment]
GitHub has sent you an invitation to join the stylelint org to whatever email your @DanielRuf uses for that, instead, maybe this will work: 

> https://github.com/stylelint/stylelint/invitations

**Comment by DanielRuf** [comment]
Ah ok. Hm, there is nothing.
bildschirmfoto 2018-05-11 um 12 42 39

**Comment by DanielRuf** [comment]
We can also move further discussions to Gitter ;-)
https://gitter.im/ntwb
https://gitter.im/DanielRuf

**Comment by ntwb** [comment]
Try this https://github.com/orgs/stylelint/invitation ?

**Comment by DanielRuf** [comment]
> Try this github.com/orgs/stylelint/invitation ?

Works, there was an invitation. Accepted it 👍 
Totally forgot the url scheme for orgs and invitations.

**Comment by ntwb** [comment]
Feel free to click the "Squash and merge" button @DanielRuf 🎉 

tl;dr: We're about to turn on automatic invites to the stylelint organisation for anyone who sends us a PR that is accepted and merged. This is just a manual kick of to ensure a few of the processes we've implemented to support this are working as intended :)

**Comment by DanielRuf** [comment]
> p.s. Welcome to the @stylelint organisation

Thanks and I'm happy to be a part and bring the project forward together (internally we plan to switch to stylelint so expect feedback and more PRs coming in the near future ;-) ) =)

**Comment by ntwb** [comment]
Woot, awesome @DanielRuf, looking forward to it 💯
